adcc/kernel/__init__.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
I think `kernel` is a bad name for this because it does not mean anything... Why not `equations`, `working_equations` or something of the sort?

Reply:
Well I thought about that, but they are not "just" equations, since imho this does not make sense (you always want some functionality like the dispatch to be close as well). Also it's not just going to be the "classic" working equations (e.g. the properties etc are also there). But I agree `kernel` is not a good name ... and in the mathematical sense of the word even wrong. But I have no better idea so far.

Reply:
Also it pops up everywhere, so this time I think it should be shorter than `working_equations` :smile:

Reply:
`core`? 😄

Reply:
Just saying: https://www.freethesaurus.com/kernel

Reply:
I'd like `substance` or `meat`... not.
But for real... I'm going to think about it more seriously...

Reply:
perform_routines? equation_factory_perform_evaluator? Yes, thanks. I don't like kernel either.

Reply:
Bikeshedding happens in #77.